var/spack/repos/builtin/packages/opencv/package.py

```diff
@@ -220,49 +220,48 @@ def cmake_args(self):
             args.extend([
                 '-DZLIB_LIBRARY_{0}:FILEPATH={1}'.format((
                     'DEBUG' if '+debug' in spec else 'RELEASE'),
```

Comment:
This package doesn't even have a `+debug` variant. It should be:
```python
if 'build_type=Debug' in spec
```

Reply:
Think this is resolved in 0a1f112. These were already in the opencv spec, I didn't think to actually check them, sorry about that.